service/src/main/kotlin/fi/espoo/evaka/messaging/daycarydailynote/DaycareDailyNoteController.kt

```diff
@@ -36,7 +36,7 @@ class DaycareDailyNoteController {
         return db.read { it.getChildDaycareDailyNotes(childId) }.let { ResponseEntity.ok(it) }
     }
 
-    @PostMapping("/child/{childId")
+    @PostMapping("/child")
```

Comment:
I have a complete overhaul of this in my branch, but merge yours first and I'll fix this when rebasing